src/display/font_loader.js

```diff
@@ -101,13 +104,16 @@ FontLoader.prototype = {
       'ABAAAAAAAAAAAD6AAAAAAAAA=='
     ));
   },
+  configurable: true
+});
 
-  addNativeFontFace: function fontLoader_addNativeFontFace(nativeFontFace) {
+  FontLoader.prototype.addNativeFontFace =
+      function fontLoader_addNativeFontFace(nativeFontFace) {
     this.nativeFontFaces.push(nativeFontFace);
     document.fonts.add(nativeFontFace);
-  },
+  };
 
-  bind: function fontLoaderBind(fonts, callback) {
+  FontLoader.prototype.bind = function fontLoaderBind(fonts, callback) {
```

Comment:
The `FontLoader.prototype` object literal was closed above, so JavaScript if-statement could be used. ("if" cannot be used inside object literal)